src/aleph/vm/models.py

```diff
@@ -81,14 +79,17 @@ class VmExecution:
     expire_task: Optional[asyncio.Task] = None
     update_task: Optional[asyncio.Task] = None
 
+    snapshot_manager: Optional[SnapshotManager]
+    systemd_manager: Optional[SystemDManager]
+
     persistent: bool = False
 
     @property
     def is_running(self) -> bool:
         return (
-            bool(self.times.starting_at and not self.times.stopping_at)
-            if not self.persistent
-            else self.systemd_manager.is_service_active(self.controller_service)
+            self.systemd_manager.is_service_active(self.controller_service)
+            if self.persistent and self.systemd_manager
+            else bool(self.times.starting_at and not self.times.stopping_at)
```

Comment:
This has been changed on main, so shouldn't be here. Be careful the next time with the merges.